### PR TITLE
add on-prem-pricing doc

### PR DESCRIPTION
**Pull request by keithhand**
taken from [old cost-model readme](https://github.com/opencost/opencost/blob/3f0185bea59e8a617b7c41e86a05a820bc07977d/README.md#how-do-you-determine-ramcpugpu-costs-for-a-node-when-this-data-isnt-provided-by-a-cloud-provider)

### CODE REVIEW

**Comment by dwbrown2** [comment]
Is there anything valuable in the OpenCost Spec that we should reference here? 

Also, thoughts on mentioning advanced pipeline for modifying these variables?

**Comment by AjayTripathy** [comment]
@jessegoodier we'd also want to update something here to reference the new GPU schema right?

**Comment by keithhand** [comment]
To separate the pulls, I can open another PR with additional docs regarding the Custom Provider Pipeline.

**Review by bstuder99 — APPROVED** [review]

**Comment by srpomeroy** [comment]
We will need to update `summary.md` for the navigation once we finalize the content and doc name.

File: on-prem-pricing.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+On-Prem Pricing
+===============
+
+How do you determine RAM/CPU/GPU costs for a node when this data isn’t provided by a cloud provider?
+---------------
+
+When explicit RAM, CPU or GPU prices are not provided by your cloud provider, the Kubecost model falls back to the ratio of base CPU, GPU and RAM price inputs supplied. The default values for these parameters are based on the marginal resource rates of the cloud provider, but they can be customized within Kubecost.
+
+These base RAM/CPU/GPU prices are normalized to ensure the sum of each component is equal to the total price of the node provisioned, based on billing rates from your provider. When the sum of RAM/CPU/GPU costs is greater (or less) than the price of the node, then the ratio between the input prices is held constant.
```
[the remark this answers is not in the document]
Comment (keithhand) [reply on-prem-pricing.md]: I updated this to "resource" in the latest commit.

File: on-prem-pricing.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+On-Prem Pricing
```
[the remark this answers is not in the document]
Comment (keithhand) [reply on-prem-pricing.md]: I updated this to "Node Pricing". I think this fits the content a bit better.

**Comment by keithhand** [comment]
I opened https://github.com/kubecost/docs/pull/459 to update the `SUMMARY.md` file.

**Pull request merged**